examples/cross-vm-communication/README.md

```diff
@@ -96,7 +96,7 @@ You are supposed to see an event. Example:
   ![The image shows how to add a network to the metamask](images/metamask-add-network.png)
 * You will be redirected to the website with the network settings.
 Click on the `Add a network manually` button at the bottom of the page. Fill in the following fields:
-  * Network name: `Golden Gate`
+  * Network name: `GGX`
```

Comment:
@akorchyn `GGX Chain` maybe :) Need to discuss and confirm

Reply:
Actually that shouldn't affect anything, I believe. That's just their local chain name.